api/src/pcapi/alembic/versions/20231102T135725_ad78bc82b4fb_populate_cipher_password_cgr_column.py

```diff
@@ -0,0 +1,30 @@
+"""
+Populate CGR `cipherPassword` column
+"""
+
+from alembic import op
+
+from pcapi import settings
+from pcapi.utils.crypto import cipher_text
+
+
+# pre/post deployment: pre
```

Comment:
Si cette migration et les 2 autres passent en **pré**-déploiement, on a un (petit) risque d'avoir le scénario suivant : 

1. La migration précédente ajoute la colonne `cipherPassword`, remplie avec NULL.
2. Cette migration remplit la colonne pour toutes les lignes.
3. Un utilisateur ajoute une ligne dans la table via le back-office (je crois ?). La colonne de cette ligne aura pour valeur NULL.
4. La migration suivante va échouer à créer la contrainte `NOT NULL`.

Évidemment c'est peu probable... mais il suffirait de changer les migrations numéro 2 et 3 en **post**-déploiement pour que tout fonctionne. Qu'en penses-tu ?

Reply:
ahaha, je pensais avoir pensé à tout, mais apparemment non 👀 Ça me va très bien en post